app/models/page.rb

```diff
@@ -43,6 +43,11 @@ def elements_hash
     hash
   end
 
+  # Return true if this page is the default page
+  def default_page?
+    return true if self == Page.default_page
```

Comment:
Line 48 can be shortened to `self == Page.default_page`

Reply:
Is there a downside to keeping it more explicit?

Reply:
There's not much of a downside to keeping it as is, if any. The very minor points in its favour are:

- `return true if self == Page.default_page` returns `true` or `nil` (never `false`). The method return type is not consistently boolean, compared to below.
- `self == Page.default_page` returns `true` or `false`. The method return type is consistently boolean, which is a little more predictable but admittedly nothing great.
- the shorter version is more conventional.